src/chesster.js

```diff
@@ -850,8 +850,8 @@ function schedulingReplyAmbiguous(bot, message){
 
 // Game has been scheduled.
 function schedulingReplyScheduled(bot, message, results, white, black) {
-    var whiteDate = results.date.clone().utcOffset(white.tz_offset/60);
-    var blackDate = results.date.clone().utcOffset(black.tz_offset/60);
+    var whiteDate = results.date.clone().tz(white.tz);
+    var blackDate = results.date.clone().tz(black.tz);
```

Comment:
Agreed. I will update the localTime Function.

I will remove the warning. I'm not 100% convinced that it's not useful, but this changeset should make chesster as accurate as he ever was in terms of time zone conversions.